deploy/docker-compose.yml

```diff
@@ -78,15 +78,42 @@ services:
         fluentd-address: "0.0.0.0:24224"
         tag: f5-do
 
+  elasticsearch:
```

Comment:
We may need to mount the data directory of elasticsearch out as a volume.

Reply:
I do not know how to reuse the data from node elasticsearch to another elasticaserach or the data volume is usable or not, you probably can have a try and investigate more after this patch. 
I think the better solution is to set up a cluster, we can have a investigate after this sprint demo.
For more secure, we can put elasticserach on a server node (physical or virtual one).